general-concepts/ebuild-revisions/text.xml

```diff
@@ -8,37 +8,99 @@
 Ebuilds may have a Gentoo revision number associated with them. This is a
 <c>-rX</c> suffix, where <c>X</c> is an integer <d/> see <uri
 link="::ebuild-writing/file-format#File Naming Rules"/>. This
-component must only be used for Gentoo changes, not upstream releases. By
-default, <c>-r0</c> is implied.
+component must only be used for Gentoo changes, not upstream releases.
+An ebuild with no explicit revision number has the implicit <c>-r0</c>
+revision.
 </p>
 
-<p>
-Ebuilds should have their <c>-rX</c> incremented whenever a change is made which
-will make a substantial difference to what gets installed by the package <d/> by
-substantial, we generally mean "something for which many users would want to
-upgrade". This is usually for bugfixes.
-For any such revision bump, the new ebuild should be based on the
-previous revision to ensure that fixes aren't dropped accidentally.
```

Comment:
IIRC we had introduced this phrase ("the new ebuild should be based on the previous revision to ensure that fixes aren't dropped accidentally") because there were some issues with ebuilds merged from overlays that were dropping previous changes. So could this sentence be kept please? Just above the examples might be a good place.